.eslintrc

```diff
@@ -1,7 +1,8 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
related to babel-eslint being [very out of date](https://stackoverflow.com/a/69557309)